frontend/src/store/FamLoginUserState.ts

```diff
@@ -150,6 +150,14 @@ const cacheUserAccess = async () => {
     }
 };
 
+const getCachedAppRoles = (application_id: number): FamRoleDto[] => {
+    const delegatedCachedData = getUserAccess()!.find(key => key.auth_key === DELEGATED_ADMIN_ROLE)?.grants.find((item) => {
```

Comment:
this should be APP_ADMIN role, so this is for APP_ADMIN to add a delegated admin

Reply:
Thanks, I'll correct this as suggested.